dev_apps/music-nga/elements/music-tab-bar.js

```diff
@@ -39,6 +42,7 @@ proto.createdCallback = function() {
   });
 
   this.selectedIndex = 0;
+  setTimeout(() => this.classList.add('resolved'), 0);
```

Comment:
I actually want to scrap these changes, was just an experiment, didint' mean to commit :yum: